src/translations_reader.py

```diff
@@ -0,0 +1,38 @@
+from src.word import Word
+from conllu import parse_incr
+from src.sentence import Sentence
+from src.translation import Translation
+
+
+def read_conllu_outputs(reference_path, hypothesis_path):
+    translations = []
+    with open(reference_path, 'r', encoding="utf-8") as ref, open(hypothesis_path, 'r', encoding="utf-8") as hyp:
+        for ref_tokenlist, hyp_tokenlist in zip(parse_incr(ref), parse_incr(hyp)):
+            ref_sentence = Sentence()
+            hyp_sentence = Sentence()
+            for token in ref_tokenlist:
+                ref_sentence.words.append(Word(token["form"], token["lemma"]))
+            for token in hyp_tokenlist:
+                hyp_sentence.words.append(Word(token["form"], token["lemma"]))
```

Comment:
it would be nice if the implementation will be more encapsulated inside the Sentence class or something like that. In example instead:
ref_sentence = Sentence()
for token in ref_tokenlist:
 ref_sentence.words.append(Word(token["form"], token["lemma"]))
something like:
ref_sentence = Sentence(ref_tokenlist)